types/chart.js/index.d.ts

```diff
@@ -416,7 +416,7 @@ declare namespace Chart {
         footerMarginTop: number;
         footerSpacing: number;
         height: number;
-        labelColors: string[];
+        labelColors: ChartTooltipLabelColor[];
```

Comment:
There's a huge difference between an array of strings and an array of objects. How did this work before?

Reply:
As far as I can see in the documentation, labelColors is just an array of strings.

Reply:
> As far as I can see in the documentation, labelColors is just an array of strings.

![image](https://user-images.githubusercontent.com/467658/87518635-cfa48280-c680-11ea-8b69-3f872a84f70f.png)

![image](https://user-images.githubusercontent.com/467658/87518703-e77c0680-c680-11ea-8649-064b672a94f7.png)

Reply:
:+1: 

So this PR should be closed in your opinion?

Reply:
Or, we could just add a Color type.

Something like:

```typescript
type Color = string
```

Then it can be used as `Color[]`

Reply:
> Or, we could just add a Color type.
> 
> Something like:
> 
> ```ts
> type Color = string
> ```
> 
> Then it can be used as `Color[]`

Yes, we could define this Color type. IMHO this is a bit confusing, but what the hell do I know...

Reply:
Fair enough. Let's keep it as it is then. `string[]`